src/background/stores/assessment-store.ts

```diff
@@ -432,12 +434,38 @@ export class AssessmentStore extends BaseStoreImpl<AssessmentStoreData> {
         testStepName: string,
         testType: VisualizationType,
     ): void {
-        const isManual = this.assessmentsProvider.getStep(testType, testStepName).isManual;
-        if (isManual !== true) {
+        const step = this.assessmentsProvider.getStep(testType, testStepName);
+        const { isManual, getInitialManualTestStatus } = step;
+
+        if (isManual) {
+            this.applyInitialManualTestStatus(
+                assessmentData,
+                testStepName,
+                testType,
+                getInitialManualTestStatus,
+            );
+        } else {
             this.updateTestStepStatusForGeneratedInstances(assessmentData, testStepName);
         }
     }
 
+    private applyInitialManualTestStatus(
+        assessmentData: AssessmentData,
+        testStepName: string,
+        testType: VisualizationType,
+        getInitialManualTestStatus: (InstanceIdToInstanceDataMap) => ManualTestStatus,
+    ): void {
+        const originalStatus = assessmentData.manualTestStepResultMap[testStepName].status;
+        if (originalStatus !== ManualTestStatus.UNKNOWN) {
+            return; // Never override an explicitly set status
```

Comment:
could this have any effect on assessments in progress? It seems not - if a status is set, we won't override it, and otherwise, it's a change we want to make (unknown -> pass)

Reply:
I think the scan complete callback that triggers this logic only happens after scans happen, so in general I'd expect it to only affect an assessment in progress if that assessment either hadn't already gotten to the requirements in question or if that assessment explicitly restarts Landmarks; I think in either case, it's desireable for the new logic to take effect